### PR TITLE
accounts for uses with incomplete profiles

### LINKED ISSUE #690: TypeError: Cannot read property 'isMember' of null
Sentry Issue: [THATUS-HQ](https://sentry.io/organizations/that-conference/issues/2332237724/?referrer=github_integration)

```
TypeError: Cannot read property 'isMember' of null
  at _ (/build/bundle.js:96:137584)
  at Object.p (/build/bundle.js:96:140513)
  at Us (/build/bundle.js:72:7121)
  at Ds (/build/bundle.js:72:6794)
```

### PR DESCRIPTION
**Pull request by csell5**
#686 didn't account for new users with incomplete profiles. They ended up in a land of nasty errors when trying to reference thatProfile from the store but it was null.

Also bumps to node 14

closes #690

### CODE REVIEW

File: src/components/home/Testimonials.svelte [hunk 1]
```diff
@@ -79,7 +79,7 @@
                   Join us today!
                 </Link>
               </p>
-            {:else if !$thatProfile.isMember}
+            {:else if !$thatProfile?.isMember}
```
Comment (brettski) [inline src/components/home/Testimonials.svelte:82]: Are we running v14 node on Vercel?

File: src/components/members/Hero.svelte [hunk 2]
```diff
@@ -38,7 +38,7 @@
         </Link>
       </div>
     </div>
-  {:else if !$thatProfile.isMember}
+  {:else if !$thatProfile?.isMember}
```
Comment (brettski) [inline src/components/members/Hero.svelte:41]: Are we running v14 node on Vercel?

**Comment by brettski** [comment]
Vercel for THAT.us is currently set to 12.x. There is a 14.x version avaiable

File: src/components/home/Testimonials.svelte (hunk 1 above)
Comment (csell5) [reply src/components/home/Testimonials.svelte:82]: svelte accounts for that when it compiles it.

File: src/components/members/Hero.svelte (hunk 2 above)
Comment (csell5) [reply src/components/members/Hero.svelte:41]: svelte accounts for that when it compiles it.

**Comment by csell5** [comment]
> Vercel for THAT.us is currently set to 12.x. There is a 14.x version avaiable

we can bump it. no reason not too.

**Comment by brettski** [comment]
Updated. I suggest, if are not already to run local with node 14.16.0

**Review by brettski — DISMISSED** [review]

**Comment by brettski** [comment]
vercel Updated

**Comment by csell5** [comment]
> Updated. I suggest, if are not already to run local with node 14.16.0

I'll update and include wth this PR, commit coming.

**Review by brettski — APPROVED** [review]

**Pull request merged**